src/components/MemoryItem.js

```diff
@@ -27,8 +27,9 @@ const Wrapper = styled.section`
 `
 
 const Image = styled.img`
-  border-radius: 20px;
   -webkit-box-shadow: 0px 10px 13px -7px #000000,
     5px 5px 15px 5px rgba(0, 0, 0, 0);
-  box-shadow: 0px 10px 13px -7px #000000, 5px 5px 15px 5px rgba(0, 0, 0, 0);
+  box-shadow: var(--img-shadow);
+  aspect-ratio: attr(width) / attr(height);
```

Comment:
You don't need to do this as modern browsers do it on their own, if you set width and height attributes for an image.